### PR TITLE
chore: added a sample that shows the use of the profanity filter

### PR DESCRIPTION
**Pull request by b-loved-dreamer**
Check out this new sample and let me know if you think changes are merited

### CODE REVIEW

File: samples/profanityFilter.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(gcsUri) {
+  // [START syncRecognizeWithProfanityFilter]
+  // Filters profanity
+
```
Comment (bcoe) [inline samples/profanityFilter.js:20]: I would add a section here that shows users what variables to uncomment:

```
  /**
   * TODO(developer): Uncomment these variables before running the sample.
   */
  // const gcsUri = 'gs://.../';
```

Comment (b-loved-dreamer) [reply samples/profanityFilter.js:20]: Done.

File: samples/system-test/profanityFilter.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const storageUri = 'gs://cloud-samples-tests/speech/brooklyn.flac';
+const text = 'how old is the Brooklyn Bridge';
+
+describe('profanityFilter', () => {
+    it('should run profanityFilter', async () => {
+      const stdout = execSync(`node profanityFilter.js ${storageUri}`)
+      assert.match(stdout, /Transcription:/ );
+    });
+  });
```
Comment (bcoe) [inline samples/system-test/profanityFilter.test.js:32]:
```suggestion
});

```

File: samples/system-test/profanityFilter.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const storageUri = 'gs://cloud-samples-tests/speech/brooklyn.flac';
+const text = 'how old is the Brooklyn Bridge';
+
+describe('profanityFilter', () => {
+    it('should run profanityFilter', async () => {
+      const stdout = execSync(`node profanityFilter.js ${storageUri}`)
+      assert.match(stdout, /Transcription:/ );
+    });
```
Comment (bcoe) [inline samples/system-test/profanityFilter.test.js:31]:
```suggestion
  it('should run profanityFilter', async () => {
    const stdout = execSync(`node profanityFilter.js ${storageUri}`)
    assert.match(stdout, /Transcription:/ );
  });
```

File: samples/system-test/profanityFilter.test.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const storageUri = 'gs://cloud-samples-tests/speech/brooklyn.flac';
+const text = 'how old is the Brooklyn Bridge';
+
+describe('profanityFilter', () => {
+  it('should run profanityFilter', async () => {
+    const stdout = execSync(`node profanityFilter.js ${storageUri}`)
+    assert.match(stdout, /Transcription:/ );
+  });
+  });
```
Comment (bcoe) [inline samples/system-test/profanityFilter.test.js:32]:
```suggestion
});
```

**Review by bcoe — APPROVED** [review]

**Pull request merged**